libproject/src/libproject.cpp

```diff
@@ -69,10 +66,10 @@ namespace Interface {
         /*virtual*/ bool open();
 
         /*!
-         * \brief Gives to user list<string> of filenames
-         * \return list<string> of filenames of project or drops exception if project wasn't loaded
+         * \brief Gives to user filepaths of files
+         * \return Files(typedef-ed container) of project or drops exception if project wasn't loaded
```

Comment:
>(typedef-ed container)
There are no need to mean such info in documentation, because it can became not container, or not typedefed...